### PR TITLE
Add test rstudio v1.0.3 tag

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

File: config/prod/tom-test-rstudio2.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ parameters:
   VpcName: "sandcastlevpc"
   KeyName: "sandbox"
   # AMIId: "ami-0810a318c4b1243c5"      # https://github.com/Sage-Bionetworks-IT/packer-base-amazonlinux2/releases/tag/v1.0.2
-  AMIId: "ami-0cd489a193a7cc7de"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.2
+  AMIId: "ami-01866cea595e4f968"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.3
```
Comment (thomasyu888) [inline config/prod/tom-test-rstudio2.yaml:20]:
```suggestion
  AMIId: "ami-0cd489a193a7cc7de"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.2
```

File: config/prod/tom-test-rstudio4.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  # AMIId: "ami-0810a318c4b1243c5"      # https://github.com/Sage-Bionetworks-IT/packer-base-amazonlinux2/releases/tag/v1.0.2
+  AMIId: "ami-0cd489a193a7cc7de"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.2
```
Comment (zaro0508) [inline config/prod/tom-test-rstudio4.yaml:20]: this isn't v1.0.3, the v1.0.3 tagged build reports `ami-01866cea595e4f968`
https://travis-ci.com/github/Sage-Bionetworks-IT/packer-rstudio/jobs/319496371#L442

File: config/prod/tom-test-rstudio4.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v2.j2
+stack_name: tom-test-rstudio2
```
Comment (thomasyu888) [inline config/prod/tom-test-rstudio4.yaml:3]:
```suggestion
stack_name: tom-test-rstudio4
```

File: config/prod/tom-test-rstudio4.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  # AMIId: "ami-0810a318c4b1243c5"      # https://github.com/Sage-Bionetworks-IT/packer-base-amazonlinux2/releases/tag/v1.0.2
+  AMIId: "ami-0cd489a193a7cc7de"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.2
```
Comment (thomasyu888) [inline config/prod/tom-test-rstudio4.yaml:20]:
```suggestion
  AMIId: "ami-01866cea595e4f968"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.3
```

**Pull request merged**